README.md

```diff
@@ -56,30 +56,30 @@ Day 1
 | Time         | Topic                                   | Instructor   |
 | ------------ | --------------------------------------- |--------------|
 | 8:30-9:00    | Setup & Intro                           |   All        |
-| 9:00-9:15    | [If it's not reproducible, it is still science?](https://github.com/UW-Madison-ACI/boot-camps/blob/2014-08-04-Davidson/BestPractices.pdf?raw=true) | TBD |
-| 9:15-10:45   | [Introducing the Shell](shell/Readme.md)| TBD |
+| 9:00-9:15    | [If it's not reproducible, it is still science?](https://github.com/UW-Madison-ACI/boot-camps/blob/2014-08-04-Davidson/BestPractices.pdf?raw=true) | Karl |
```

Comment:
Should probably drop this link and consider renaming per @kbroman before going live